package.json

```diff
@@ -86,7 +86,7 @@
     "rimraf": "~2.6.2",
     "selenium-webdriver": "3.6.0",
     "sinon": "~4.1.3",
-    "typescript": "3.4.5"
+    "typescript": "3.8.3"
```

Comment:
Do we want to change the version, we are mostly trying to make the package forwards compatible. Is that possible?

Reply:
Oh that was just my misunderstanding. I've reverted the package version, still builds fine against 3.4.5